student.rb

```diff
@@ -0,0 +1,11 @@
+class Student < Person
+  def initialize(age, play_hooky, name: 'unknown', parent_permission: true, classroom: '10th')
+    super(age, name, parent_permission)
+    @classroom = classroom
+    @play_hooky = play_hooky
```

Comment:
- Hey, since the last reviewer didn't raise this as a requirement I am marking this as optional. However, I recommend you implement this.
Consider removing play_hooky variable from the constructor, since this variable is not needed anywhere throughout the project.

Reply:
Implemented 👍